From 11e8582fe70dc6d0006a34fea4380c0eedadbb15 Mon Sep 17 00:00:00 2001
From: Sultanxda <sultanxda@gmail.com>
Date: Thu, 5 Nov 2015 22:15:07 -0800
Subject: [PATCH 058/194] SnapdragonCamera: Remove 'off' option for antibanding

Antibanding is very important for UX with regard to photographing in
artificial lighting, so don't provide users with an option to turn
antibanding off.

Change-Id: I080a48a88b6faee69a67df8706e0a3acd1c233f0
---
 res/values/qcomarrays.xml  | 4 +---
 res/values/qcomstrings.xml | 3 +--
 2 files changed, 2 insertions(+), 5 deletions(-)

diff --git a/res/values/qcomarrays.xml b/res/values/qcomarrays.xml
index 4285c5097..34212e311 100755
--- a/res/values/qcomarrays.xml
+++ b/res/values/qcomarrays.xml
@@ -448,14 +448,12 @@
          <item>@string/pref_camera_antibanding_entry_0</item>
          <item>@string/pref_camera_antibanding_entry_1</item>
          <item>@string/pref_camera_antibanding_entry_2</item>
-         <item>@string/pref_camera_antibanding_entry_3</item>
      </string-array>
 
      <string-array name="pref_camera_antibanding_entryvalues">
-         <item>off</item>
+         <item>auto</item>
          <item>50hz</item>
          <item>60hz</item>
-         <item>auto</item>
      </string-array>
 
      <!-- Camera Preferences Histogram dialog box entries -->
diff --git a/res/values/qcomstrings.xml b/res/values/qcomstrings.xml
index e2505bce3..a663af01f 100755
--- a/res/values/qcomstrings.xml
+++ b/res/values/qcomstrings.xml
@@ -528,10 +528,9 @@
      <string name="pref_camera_antibanding_default">auto</string>
 
      <!-- Settings screen, Anti Banding entries -->
-     <string name="pref_camera_antibanding_entry_0">Off</string>
+     <string name="pref_camera_antibanding_entry_0">Auto</string>
      <string name="pref_camera_antibanding_entry_1">50 Hz (Europe)</string>
      <string name="pref_camera_antibanding_entry_2">60 Hz (USA)</string>
-     <string name="pref_camera_antibanding_entry_3">Auto</string>
 
         <!-- Default color effect setting. -->
     <string name="pref_camera_coloreffect_default" translatable="false">none</string>
-- 
2.17.1

